src/domain/shared/components/ShareDialog/ShareDialog.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+import { Dialog } from '@mui/material';
+import { DialogContent, DialogTitle } from '../../../../common/components/core/dialog';
+import { ShareComponent, ShareComponentTitle, ShareComponentProps } from './ShareComponent';
+
+interface ShareDialogProps extends ShareComponentProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+export const ShareDialog: FC<ShareDialogProps> = ({ open, onClose, ...props }) => {
+  return (
+    <Dialog open={open}>
```

Comment:
define the `onClose` property on the dialog so `ESC` and clicking outside closes it.